app/locales/nl/translations.js

```diff
@@ -179,8 +179,7 @@ export default {
     },
     lustrum: {
       lustrumBar: {
-        signUp: 'Schrijf je in!',
-        signedUp: 'Ingeschreven!'
+        signUp: 'Schrijf je in via de mail!',
```

Comment:
Doen we niet gewoon google form? Dat heb ik veel liever. En dan is de kreet "Schrijf je in!" veel logischer

Reply:
Ja dat is wel waar